steps/cadence-genus-synthesis/scripts/designer-interface.tcl

```diff
@@ -56,10 +56,7 @@ set vars(adk_dir) inputs/adk
 
 set vars(libs_typical,timing) \
     [join "
-        $vars(adk_dir)/stdcells.lib
-        [lsort [glob -nocomplain $vars(adk_dir)/stdcells-lvt.lib]]
-        [lsort [glob -nocomplain $vars(adk_dir)/stdcells-ulvt.lib]]
-        [lsort [glob -nocomplain $vars(adk_dir)/stdcells-pm.lib]]
+        [lsort [glob -nocomplain $vars(adk_dir)/stdcells*.lib]]
```

Comment:
Thanks Steve for catching these issues! We were trying to avoid hard-coding to make the nodes more general as we worked on the Intel bring up. Maybe we can revert this pull request and rethink how to do this?

Reply:
Does changing the order of the library list cause any issues? For the second problem, it looks like we would need to think about the convention for naming the bc files to ensure they are not picked up unintentionally.

Reply:
> Thanks Steve for catching these issues! We were trying to avoid hard-coding to make the nodes more general as we worked on the Intel bring up. Maybe we can revert this pull request and rethink how to do this?

Yes I agree there should be a better way, we just need to think about it a little bit. Meanwhile, I suppose the right thing is to revert, if that's easy to do...otherwise we can just wait until the better solution arrives, hopefully soon...

Reply:
> Does changing the order of the library list cause any issues? For the second problem, it looks like we would need to think about the convention for naming the bc files to ensure they are not picked up unintentionally.

In the past it has caused issues, and certainly if you design with one order and then the order changes, unexpected things happen, i.e. you thought you were going to get an LVT cell and you get a ULVT cell etc. Once an order is established and the kinks are addressed, I suppose it should be fine going forward. If it were just me, being a big fan of preserving backward compatibility, I'd probably do something terrible like `if ($::env(NEED_BACKWARD_COMPATIBILITY_ON_STDLIB) { emit-warning-and-do-old-terrible-thing } else { do-elegant-new thing }`

Reply:
I agree with reverting this PR and creating a new one (and will add a backwards compatibility check in the new one). Here is the revert PR: https://github.com/mflowgen/mflowgen/pull/132. I do not think I have permissions to merge it in, so if you or Chris or Alex could do that, that would be great, thanks!